easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_80.eb

```diff
@@ -15,22 +19,19 @@ description = """ MuTect is a method developed at the Broad Institute for the re
 
 toolchain = {'name': 'dummy', 'version': ''}
 
-source_urls = ['http://www.broadinstitute.org/cancer/cga/sites/default/files/data/tools/mutect/']
+# Need to be logged in to download, at this URL
+# source_urls = ['http://www.broadinstitute.org/cancer/cga/mutect_download']
```

Comment:
I would just go with `# http://www.broadinstitute.org/cancer/cga/mutect_download`, dropping `source_urls`, since even if registration would not be needed the `source_urls` spec would still be wrong...